build-tools/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchCluster.java

```diff
@@ -390,6 +390,11 @@ public void user(Map<String, String> userSpec) {
         nodes.all(node -> node.user(userSpec));
     }
 
+    @Override
+    public void securityRoles(File rolesYml) {
```

Comment:
Could we just call this `rolesFile` or `securityRolesFile` to be more descriptive?